src/shims/foreign_items.rs

```diff
@@ -364,24 +312,6 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
                 this.write_scalar(Scalar::from_int(result, Size::from_bits(32)), dest)?;
             }
 
-            "memrchr" => {
```

Comment:
Why move memrchr but not memchr? That's odd.

Reply:
I think this one should stay here.

Reply:
I couldn't find a manual entry for `memrchar` on windows.

Reply:
I'd stay still keep it with `memchr` -- it just belongs together thematically, and also the implementation should work just fine on Windows.